games/sprig_dodge.js

```diff
@@ -0,0 +1,180 @@
+/*
+@title: sprig_dodging
+@author: sam liu and lucas
+
+Instructions:
+
+Welcome to Sprig!!!
+
+Hit "run" to execute the code and
+start the game (you can also press shift+enter).
+
+To beat each level you'll have to edit the code.
+
+The code for this game starts below this comment.
+
+Made based on this workshop https://workshops.hackclub.com/sprig_dodge/
+
+The goal is to dodge the fireballs.
+
+Click the "open help" to discover your toolkit.
+
+--------
+Level 1
+--------
+
+Add the player and obstacles in map
```

Comment:
```suggestion
Add the player to the map
```
Only the player is needed as fireballs are generated